modules/cabal-project.nix

```diff
@@ -14,6 +14,15 @@ in {
     compiler-nix-name = mkOption {
       type = str;
       description = "The name of the ghc compiler to use eg. \"ghc884\"";
+      # Map short version names to the latest GHC version.
+      # TODO: perhaps combine this with the `latestVer` mapping in `overlays/boostrap.nix`.
```

Comment:
Yes, if we could _automate_ this selection, that would be rad!

Reply:
I'm going to leave it for another PR.